src/seq/nucleotide.jl

```diff
@@ -123,6 +123,16 @@ function ispyrimidine(nt::Nucleotide)
 end
 
 """
+    isppambiguous(nt::Nucleotide)
+
+Test if nucleotide could be either a Purine or a Pyrimidine.
+"""
+function isppambiguous(nt::Nucleotide)
```

Comment:
I think this is something accidentally left in from when I was trying to be clever and use all the possible ambiguity codes before I decided/realised that doing so was a stupid idea (expanded in first comment). I'll remove this in the next commit.